src/OpenFga.Sdk/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,16 @@
+//
```

Comment:
Replaces `OpenFga.Sdk.csproject` ItemGroup as a best practice approach toward the same end.